src/content_cao.cpp

```diff
@@ -1492,7 +1492,14 @@ void GenericCAO::updateAnimation()
 
 	if (m_animated_meshnode->getStartFrame() != m_animation_range.X ||
 		m_animated_meshnode->getEndFrame() != m_animation_range.Y)
+                {
```

Comment:
Please use tabs to intend. Also move the bracket up, so it looks like `m_animation_range.Y) {`